libr/bin/format/mach0/mach0.c

```diff
@@ -2993,22 +2993,22 @@ static void _parse_symbols(RBinFile *bf, struct MACH0_(obj_t) *bin, HtPP *symcac
 		}
 
 		from = R_MIN (from, symbols_size / sizeof (RBinSymbol));
-		to = R_MIN (R_MIN (to, bin->nsymtab), symbols_size / sizeof (RBinSymbol));
+		to = R_MIN (R_MIN (to, mo->nsymtab), symbols_size / sizeof (RBinSymbol));
 		ut32 maxsymbols = symbols_size / sizeof (RBinSymbol);
 		if (symbols_count >= maxsymbols) {
 			symbols_count = maxsymbols - 1;
 			R_LOG_WARN ("Truncated symbol table");
 		}
 
 		for (i = from; i < to && j < symbols_count; i++, j++) {
```

Comment:
I think you can do a `reserve` call to the vector here, to avoid re-allocating all the time